src/odemis/gui/cont/stream.py

```diff
@@ -1392,10 +1391,11 @@ def _create_immersion_mode_cbox(self):
         # HACK manually create the checkbox using custom gb_sizer's pos and span
         _ = self.stream_panel._add_side_label("Immersion Mode")
         cbox_immersion_mode = wx.CheckBox(self.stream_panel._panel, wx.ID_ANY,
-                                 style=wx.ALIGN_RIGHT | wx.NO_BORDER)
+                                          style=wx.ALIGN_RIGHT | wx.NO_BORDER)
 
         self.stream_panel.gb_sizer.Add(cbox_immersion_mode, (self.stream_panel.num_rows, 2), span=(1, 1),
-                          flag=wx.ALIGN_CENTRE_VERTICAL | wx.EXPAND | wx.TOP | wx.BOTTOM, border=5)
+                                       flag=wx.ALIGN_CENTRE_VERTICAL | wx.EXPAND | wx.TOP | wx.BOTTOM,
+                                       border=5)
```

Comment:
These changes should be in the other commit if you want to keep two separate commits